ec_api/apps/api_proxy/tests/response_builder.py

```diff
@@ -0,0 +1,256 @@
+"""
+A set of tools for creating responses from the DC aggregator API.
+
+Used for testing.
+
+"""
+import datetime
+import json
+from random import randrange
+
+from faker import Faker
+from uk_election_ids.election_ids import IdBuilder
+
+from api_proxy.tests.name_factory import make_ward_name, make_org_name
+
+
+class BaseResponse:
+    def __init__(self):
+        self.options = {}
+        self._response = self.get_response()
```

Comment:
Wondering if this is needed as `self._response` seems to get build when `.response` is used on a subclass

Reply:
See my other comment about this being the initial response - we need to know that `self._response` exists all the time, with at least the data in `get_response`. I'll push a change to rename this to `get_initial_response`